src/it/release-start-finish-2-it/expectations/develop/expected-pom.xml

```diff
@@ -0,0 +1,22 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.amashchenko.maven.plugin</groupId>
+    <artifactId>gitflow-maven-test</artifactId>
+    <packaging>pom</packaging>
+    <version>0.0.4-SNAPSHOT</version>
+    <modules>
+        <module>withAgregatorAsParent</module>
+        <module>withoutAgregatorAsParent</module>
+    </modules>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.amashchenko.maven.plugin</groupId>
+                <artifactId>withoutAgregatorAsParent</artifactId>
+                <version>${project.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+</project>
```

Comment:
Nitpick: Some POMs end with a newline (like this), others don't (like `src/it/release-start-finish-2-it/expectations/develop/withAgregatorAsParent/expected-pom.xml`).

Reply:
also some pom.xml files hat an indentation with of only two spaces. Fixed that too.